homeassistant/components/media_player/__init__.py

```diff
@@ -474,6 +477,11 @@ def source(self):
         return None
 
     @property
+    def source_list(self):
+        """List of available input sources."""
+        return []
```

Comment:
One last thing, let's default to None so that we won't add it to the attributes if not defined